pkg/models/v1/models.go

```diff
@@ -176,6 +176,67 @@ func AppendAdminEmailLog(vasp *pb.VASP, reason string, subject string) (err erro
 	return nil
 }
 
+// VASPEmailEntry represents a log entry relevant to a single contact. It embeds an
+// email log entry and adds contact-specific information so that higher level methods
+// reading a unified contact log can distinguish between entries for different contacts.
+// TODO: This metadata should really be stored in the email log entry itself.
+type VASPEmailEntry struct {
+	*EmailLogEntry
+	ContactType string
+}
```

Comment:
I'm not especially proud of this but the email log currently doesn't contain the recipient email address or contact type, which we may have overlooked. In order to make this actually useful, I'm currently adding the contact type as a separate field in the response. This will unblock the current frontend work but should we create a story to add some more identifying info to the email log entries and another one to write the migration?

Reply:
Yes, let's create the story to add more identifying information and perform a migration; thank you for adding this temporary work around!